website/docs/whats-new/release-notes/partials/components.md

```diff
@@ -0,0 +1,200 @@
+
+- 🔄 Updated dependencies:
+    - @hashicorp/ember-flight-icons@3.1.1
+
+## 2.10.0
+
+**Month Day, Year**
+
+- `FileInput` component added. [(#1535)](https://github.com/hashicorp/design-system/pull/1535) Thanks [@KristinLBradley](https://github.com/KristinLBradley)!
+
+- `hds/copy/index.js` removed unnecessary export. [(#1570)](https://github.com/hashicorp/design-system/pull/1570) Thanks [@Dhaulagiri](https://github.com/Dhaulagiri)! 
+
+- `Hds::Tabs` fix missing tab indicator when used in Modal or Flyout [(#1575)](https://github.com/hashicorp/design-system/pull/1575) Thanks [@alex-ju](https://github.com/alex-ju)! 
+
+- `Copy::Snippet` add support for container in and update API docs. [(#1567)](https://github.com/hashicorp/design-system/pull/1567) Thanks [@MelSumner](https://github.com/MelSumner)!
+
+- `Hds::Tabs` - Fix tabs and panels misbehaving on route change. [(#1571)](https://github.com/hashicorp/design-system/pull/1571) Thanks [@alex-ju](https://github.com/alex-ju)!
+
+- `error icon` update for copy components. [(#1568)](https://github.com/hashicorp/design-system/pull/1568) Thanks [@MelSumner](https://github.com/MelSumner)! 
+
+- `Hds::SegmentedGroup` prevent `border-radius` from interfering with underlying elements. [(#1555)](https://github.com/hashicorp/design-system/pull/1555) Thanks [@alex-ju](https://github.com/alex-ju)! 
+
+- `ThSort` resolved issue where ThSort was not supporting right-aligned text properly [(#1565)](https://github.com/hashicorp/design-system/pull/1565) Thanks [@MelSumner](https://github.com/MelSumner)!
+
+- 🔄 Updated dependencies:
+    - @hashicorp/ember-flight-icons@3.1.0
+
+## 2.9.0
+
+**Month Day, Year**
+
+- `Hds::CopyButton` and `Hds::CopySnippet` components added. [(#1488)](https://github.com/hashicorp/design-system/pull/1488) Thanks [@MelSumner](https://github.com/MelSumner)!
+
+- `copywrite headers` adjusted closing brace to avoid terminal noise [(#1539)](https://github.com/hashicorp/design-system/pull/1539) Thanks [@Dhaulagiri](https://github.com/Dhaulagiri)! 
+
+- `Hds::Modal` reduce test flakiness around closing Modal when using `@ember/test-helpers` [(#1549)](https://github.com/hashicorp/design-system/pull/1549) Thanks [@fivetanley](https://github.com/fivetanley)! 
+
+- `Embroider warnings` fix caused by incorrect export of internal utility functions. [(#1530)](https://github.com/hashicorp/design-system/pull/1530) Thanks [@alex-ju](https://github.com/alex-ju)! 
+
+- `Hds::Flyout` - reduce test flakiness around closing Flyout when using `@ember/test-helpers`. [(#1552)](https://github.com/hashicorp/design-system/pull/1552) Thanks [@alex-ju](https://github.com/alex-ju)! 
+
+- `@layout` fix error message for unexpected values. [(#1529)](https://github.com/hashicorp/design-system/pull/1529) Thanks [@alex-ju](https://github.com/alex-ju)!  
+
+- 🔄 Updated dependencies:
+    - @hashicorp/ember-flight-icons@3.0.9
```

Comment:
```suggestion
**August 7, 2023**

- `Hds::CopyButton` and `Hds::CopySnippet` - components added. [#1488](https://github.com/hashicorp/design-system/pull/1488) Thanks [@MelSumner](https://github.com/MelSumner)!

- Adjusted closing brace on copywrite headers to avoid terminal noise. [#1539](https://github.com/hashicorp/design-system/pull/1539) Thanks [@Dhaulagiri](https://github.com/Dhaulagiri)! 

- `Hds::Modal` - reduced test flakiness around closing Modal when using `@ember/test-helpers` [#1549](https://github.com/hashicorp/design-system/pull/1549) Thanks [@fivetanley](https://github.com/fivetanley)! 

- Fix Embroider warnings caused by incorrect export of internal utility functions. [#1530](https://github.com/hashicorp/design-system/pull/1530) Thanks [@alex-ju](https://github.com/alex-ju)! 

- `Hds::Flyout` - reduced test flakiness around closing Flyout when using `@ember/test-helpers`. [#1552](https://github.com/hashicorp/design-system/pull/1552) Thanks [@alex-ju](https://github.com/alex-ju)! 

- `Hds::Form::Field` - fixed error message for unexpected `@layout` values. [#1529](https://github.com/hashicorp/design-system/pull/1529) Thanks [@alex-ju](https://github.com/alex-ju)!  

- 🔄 Updated dependencies:
    - @hashicorp/ember-flight-icons@3.0.9
```